devel/jenkins/Portfile

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           java 1.0
+
+name                jenkins
+version             2.401.3
+revision            0
+
+homepage            https://jenkins.io
+
+description         The leading open source automation server,\
+                    Jenkins provides hundreds of plugins to support building,\
+                    deploying and automating any project.
+
+long_description    ${description}
+
+categories          devel java
+license             MIT
+
+maintainers         {gmail.com:slashapp @slashapp} openmaintainer
+
+master_sites        https://get.jenkins.io/war-stable/${version}/
+distfiles           ${name}.war
+
+checksums           rmd160  0a762953129148086beced9450a364467a136b40 \
+                    sha256  a798a0c5481a8ffb0320d9121f6cf49dc575c369028daae17a4dd398b69e000d \
+                    size    98398119
+
+java.version        11+
+java.fallback       openjdk11
+
+use_configure       no
+
+extract {}
+
+build {}
+
+set jenkins_launch_dir  /Library/LaunchDaemons
+set jenkins_plist_dir   ${prefix}/etc/LaunchDaemons/org.macports.${name}
+set jenkins_plist       org.macports.${name}.plist
+set jenkins_home_dir    ${prefix}/var/lib/${name}
+set jenkins_exec_dir    ${prefix}/libexec/${name}
+set jenkins_exec        ${jenkins_exec_dir}/${distfiles}
+set jenkins_host        127.0.0.1
+set jenkins_port        8080
```

Comment:
Ports shouldn't install their own launch agent/daemon. Instead, use the `startupitem` functionality, documented here:

https://guide.macports.org/#reference.startupitems

It will also help simplify your port.

Reply:
> Ports shouldn't install their own launch agent/daemon. Instead, use the `startupitem` functionality, documented here:
> 
> [guide.macports.org/#reference.startupitems](https://guide.macports.org/#reference.startupitems)
> 
> It will also help simplify your port.

Though you'll still want to supply the launchd plist, and specify that via `startupitem.custom_file`.

Reply:
startupitem.custom_file added from MacPorts 2.8, According you suggestion, I will change plist to startupitem.create.